src/TLuaInterpreter.cpp

```diff
@@ -3265,202 +3265,322 @@ int TLuaInterpreter::setBackgroundImage( lua_State *L )
 
 int TLuaInterpreter::setLabelClickCallback( lua_State *L )
 {
-    string luaSendText="";
-    if( ! lua_isstring( L, 1 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if(! pHost) {
+        lua_pushstring( L, tr( "setLabelClickCallback: NULL Host pointer - something is wrong!" )
+                        .toUtf8().constData() );
         lua_error( L );
         return 1;
     }
-    else
-    {
-        luaSendText = lua_tostring( L, 1 );
-    }
-    string luaName="";
-    if( ! lua_isstring( L, 2 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+
+    QString labelName;
+    if( ! lua_isstring( L, 1 ) ) {
+        lua_pushstring( L, tr( "setLabelClickCallback: bad argument #1 type (label name as string expected, got %1!)" )
```

Comment:
I'll try to make this my last comment on this sub-topic, but the whole idea about hash lookups ___was___ for them to be fast or so I thought! It is easier for me as a coder to formulate as much of text stuff that _might_ need to be translated as I develop it and then comment it as something that might not need it rather than have to revisit it later and insert the coding than - it is stronger in my mind in the first case and should something later prove to not be required to go through translation the, now somewhat depreciated, `QStringLiteral` will work as a temporary means to remove such items fairly simply until a better but more convoluted `QLatin1String` form can be made!